src/Logic/operate.js

```diff
@@ -30,7 +30,7 @@ const Operate = (number1, number2, operation) => {
       }
       break;
     case '%':
-      results = numberTwo.div(100);
+      results = numberOne.div(100);
```

Comment:
- [ ] [REQUIRED] I got an error related to the prop-types on my console when I perform this operation (%). please fix it
![1](https://user-images.githubusercontent.com/58552748/128648894-5c0d9baf-44c0-4045-9455-ce7ec5dc1591.gif)